# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/md-5/tests/data/md5.blb
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 third_party/rust/md-5/tests/data/md5.blb
Binary file third_party/rust/md-5/tests/data/md5.blb has changed